src/functions/env.ts

```diff
@@ -111,6 +120,9 @@ export async function addEnvs(
   projectId: string,
   envs: Record<string, string>
 ): Promise<Record<string, string>> {
+  for (const [key] of Object.entries(envs)) {
```

Comment:
Is there a reason to use Object.entries instead of Object.keys here? I feel like this is cleaner as:
```
for (const key of Object.keys(envs)) {
```

Reply:
Good call. Will fix.